tests/configs/semantic_map-config.yml

```diff
@@ -0,0 +1,8 @@
+language: en
+
+pipeline:
+- name: WhitespaceTokenizer
+- name: rasa_nlu_examples.featurizers.sparse.SemanticMapFeaturizer
```

Comment:
Let's maybe also add the countvectorizers here. Otherwise, this smoke test doesn't represent that starting point that a user might have.